src/SUFST/src/xbee_comms.c

```diff
@@ -0,0 +1,170 @@
+#include <tx_api.h>
+#include "rtcan.h"
+#include "xbee_comms.h"
+#include "usart.h"
+
+/* Include transmit frame API layer */
+#include "xbee/wpan.h"
+#include "xbee/device.h"
+#include "wpan/aps.h"
+#include "xbee/delivery_status.h"
+
+
+#define QUEUE_SEND_THREAD_PRIORITY             10
```

Comment:
Constants named correctly? I think this should have prefix `XBEE_COMMS_` rather than `QUEUE_SEND_`.